### DISCUSSION TITLE
UHF in xtb

### DISCUSSION THREAD

**Discussion by WentongZhou** [grimme-lab/xtb#792] [Q&A] [answered]
Hi there,

I have one question about the `--uhf` in `xtb`. Most other programs use multiplicity to represent the electron spin information. But the definition here of `uhf` is a little different. it is number of unpaired electrons
When I tried to use `xtb` and `xtbiff` to generate interactions between CH3+ and indole. It seems that the E_ES_atom is high in positive. Is it reasonable?
`xtb ch3+.xyz --chrg 1 --uhf 0 --lmo`\
`rm xtblmoinfo 1`\
`xtb benzene.xyz --chrg 0 --uhf 0 --lmo`\
`rm xtblmoinfo 2`\
`xtbiff 1 2 -sp`\
By using this method, I got the information below (kcal/mol):
E Pauli : 3.132
E disp ATM: 0.001
E disp 2B: -1.430
E disp total : -1.428
E ES atom: 17.383
E ES LMO: 0.223
E ES total: 17.607
E induction: 0.022
E CT: -101.289
Eint total,gas: -81.957

![image](https://user-images.githubusercontent.com/64748721/227224643-df68ebb8-0996-4f46-98f4-0e0813ca2543.png)

**Comment by cplett** [comment, marked as answer]
Hi, 
Indeed,  `uhf` is the number of unpaired electrons so defining `--uhf 0` is correct in this case.
If the two molecules come quite close (which I would expect due to the positive charge of the CH3 fragment), the electrostatic energy can become really high due to the electron/electron repulsion and is compensated by the charge transfer term. Therefore, the result seems reasonable.
